src/dataviews/category-dataview-model.js

```diff
@@ -152,7 +151,7 @@ module.exports = DataviewModelBase.extend({
   },
 
   getCount: function () {
-    return this.get('allCategoryNames').length;
+    return this.get('categoriesCount');
```

Comment:
That's we had originally: https://github.com/CartoDB/cartodb.js/blob/v4/src/dataviews/category-dataview-model.js#L155